src/app/backend/resource/common/controller.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 The Kubernetes Dashboard Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// IsControlledBy checks if the  object has a controllerRef set to the given owner.
+func IsControlledBy(obj v1.Object, owner v1.Object) bool {
+	ref := GetControllerOf(obj)
+	if ref == nil {
+		return false
+	}
+	return ref.UID == owner.GetUID()
+}
+
+// GetControllerOf returns a pointer to a copy of the controllerRef if controllee has a controller.
+func GetControllerOf(controllee v1.Object) *v1.OwnerReference {
```

Comment:
It seems that dashboard repo removes main repo (`/kubernetes`) dependencies a few hours ago. Otherwise, we are able to reuse the function declared in the main repo. Any reason for the dependency removal?